ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_slovakia_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiODQyMjMzNzA0LjE3NDYzMTgyNDQifSwiYW5vbklkIjoiZTM1MTA4ZDI3Njc2YWRlNCJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5zay9wYXNzZW5nZXJjYXJzL21vZGVscy9zdXYvZ2xjL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYxIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMTA4Mzk5MzkxOTEwNTY0NDciLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5637e3c9a75a <unknown>
#1 0x5637e374d4b0 <unknown>
#2 0x5637e3733d86 <unknown>
#3 0x5637e373269a <unknown>
#4 0x5637e3732dff <unknown>
#5 0x5637e3732d54 <unknown>
#6 0x5637e375169f <unknown>
#7 0x5637e37f692a <unknown>
#8 0x5637e37c4592 <unknown>
#9 0x5637e37ea5e0 <unknown>
#10 0x5637e37c4363 <unknown>
#11 0x5637e3790d63 <unknown>
#12 0x5637e37919c1 <unknown>
#13 0x5637e3c5fa6b <unknown>
#14 0x5637e3c63951 <unknown>
#15 0x5637e3c47b62 <unknown>
#16 0x5637e3c644c4 <unknown>
#17 0x5637e3c2c13f <unknown>
#18 0x5637e3c886f8 <unknown>
#19 0x5637e3c888d6 <unknown>
#20 0x5637e3c995a6 <unknown>
#21 0x7fea1e29caa4 <unknown>
#22 0x7fea1e329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_slovakia_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6Ijg0MjIzMzcwNC4xNzQ2MzE4MjQ0In0sImFub25JZCI6ImUzNTEwOGQyNzY3NmFkZTQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouc2svcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjE4NTQ0MjQ2ODgxOTAxMTcyIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5637e3c9a75a <unknown>
#1 0x5637e374d4b0 <unknown>
#2 0x5637e3733d86 <unknown>
#3 0x5637e373269a <unknown>
#4 0x5637e3732dff <unknown>
#5 0x5637e3732d54 <unknown>
#6 0x5637e375169f <unknown>
#7 0x5637e37f692a <unknown>
#8 0x5637e37c4592 <unknown>
#9 0x5637e37ea5e0 <unknown>
#10 0x5637e37c4363 <unknown>
#11 0x5637e3790d63 <unknown>
#12 0x5637e37919c1 <unknown>
#13 0x5637e3c5fa6b <unknown>
#14 0x5637e3c63951 <unknown>
#15 0x5637e3c47b62 <unknown>
#16 0x5637e3c644c4 <unknown>
#17 0x5637e3c2c13f <unknown>
#18 0x5637e3c886f8 <unknown>
#19 0x5637e3c888d6 <unknown>
#20 0x5637e3c995a6 <unknown>
#21 0x7fea1e29caa4 <unknown>
#22 0x7fea1e329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_slovakia_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyNTQ2IiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJza19TSyIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnNrL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjEifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6Ijg0MjIzMzcwNC4xNzQ2MzE4MjQ0In0sImFub25JZCI6ImUzNTEwOGQyNzY3NmFkZTQifSwicGVyZm9ybWFuY2UiOnt9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7IlByb2R1Y3QiOnsiX2lkIjoiMjU0NiIsImRpbWVuc2lvbnMiOnsiQmF1bXVzdGVyU2hvcnQiOlsiMjU0NiJdfX19LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjEwMDk0ODc4OTg2MTI0ODExIn0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5637e3c9a75a <unknown>
#1 0x5637e374d4b0 <unknown>
#2 0x5637e3733d86 <unknown>
#3 0x5637e373269a <unknown>
#4 0x5637e3732dff <unknown>
#5 0x5637e3732d54 <unknown>
#6 0x5637e375169f <unknown>
#7 0x5637e37f692a <unknown>
#8 0x5637e37c4592 <unknown>
#9 0x5637e37ea5e0 <unknown>
#10 0x5637e37c4363 <unknown>
#11 0x5637e3790d63 <unknown>
#12 0x5637e37919c1 <unknown>
#13 0x5637e3c5fa6b <unknown>
#14 0x5637e3c63951 <unknown>
#15 0x5637e3c47b62 <unknown>
#16 0x5637e3c644c4 <unknown>
#17 0x5637e3c2c13f <unknown>
#18 0x5637e3c886f8 <unknown>
#19 0x5637e3c888d6 <unknown>
#20 0x5637e3c995a6 <unknown>
#21 0x7fea1e29caa4 <unknown>
#22 0x7fea1e329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_slovakia_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnNrL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInZlaGljbGVDb2RlIjoieDI1NCIsInZlaGljbGVCTSI6IjI1NDYiLCJ2ZWhpY2xlUElIaXN0b3J5IjoiMjU0NiIsInRpbWVzdGFtcCI6MTc0NjMxODI0ODc5MiwiZXhwaXJhdGlvbiI6MTIwfX19LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6Ijg0MjIzMzcwNC4xNzQ2MzE4MjQ0In0sImFub25JZCI6ImUzNTEwOGQyNzY3NmFkZTQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouc2svcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sIl90b29sc0V2ZW50TGlua0lkIjoiNTg2NDczODA4OTE2NjcwMSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5637e3c9a75a <unknown>
#1 0x5637e374d4b0 <unknown>
#2 0x5637e3733d86 <unknown>
#3 0x5637e373269a <unknown>
#4 0x5637e3732dff <unknown>
#5 0x5637e3732d54 <unknown>
#6 0x5637e375169f <unknown>
#7 0x5637e37f692a <unknown>
#8 0x5637e37c4592 <unknown>
#9 0x5637e37ea5e0 <unknown>
#10 0x5637e37c4363 <unknown>
#11 0x5637e3790d63 <unknown>
#12 0x5637e37919c1 <unknown>
#13 0x5637e3c5fa6b <unknown>
#14 0x5637e3c63951 <unknown>
#15 0x5637e3c47b62 <unknown>
#16 0x5637e3c644c4 <unknown>
#17 0x5637e3c2c13f <unknown>
#18 0x5637e3c886f8 <unknown>
#19 0x5637e3c888d6 <unknown>
#20 0x5637e3c995a6 <unknown>
#21 0x7fea1e29caa4 <unknown>
#22 0x7fea1e329c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'